.gitignore

```diff
@@ -69,3 +69,5 @@ be/tags
 !.clang-format
 !.github
 build/
+cmake-build-debug/
+CMakeLists.txt
```

Comment:
Why ignore CMakeLists.txt?

Reply:
The file at the root folder just is added to index files for Clion. ignoring it avoids adding it to PRs by accident.